src/More/Licenses.tsx

```diff
@@ -20,12 +20,16 @@ const Licenses: FunctionComponent = () => {
   const { t } = useTranslation()
   const { applicationName } = useApplicationName()
 
-  const infoEmailAddress = "info@pathcheck.org"
-  const infoEmailLink = "mailto:info@pathcheck.org"
-  const pathCheckWebAddress = "pathcheck.org"
-  const pathCheckUrl = "https://pathcheck.org/"
   const privacyPolicyUrl =
     env.PRIVACY_POLICY_URL || "https://pathcheck.org/privacy-policy/"
+  const healthAuthorityName =
+    env.GAEN_AUTHORITY_NAME || "PathCheck Organization"
+  const haAddressLine1 = env.GAEN_AUTHORITY_ADDRESS_LINE_1 || "58 Day Street"
+  const haAddressLine2 = env.GAEN_AUTHORITY_ADDRESS_LINE_2 || "Box 441621"
+  const haAddressLine3 =
+    env.GAEN_AUTHORITY_ADDRESS_LINE_3 || "Somerville, MA 02144"
+  const haAddressLine4 = env.GAEN_AUTHORITY_ADDRESS_LINE_4 || "USA"
```

Comment:
I'm kind of worried about this but we will not solve international addresses when the moment comes that a different country won't have all the components we have here. Probably this ship sailed but did we consider having one address with `\n` to format it?

Reply:
Yeah we're actually removing all this stuff anyways but that's a good point!